apps/admin-server/src/hooks/use-user.tsx

```diff
@@ -2,24 +2,48 @@ import { useRouter } from 'next/router';
 import useSWR from 'swr';
 
 export default function useUser() {
+
   const router = useRouter();
-  const userId = router.query.user;
-  const url = `/api/openstad/api/project/1/user/${userId}`;
+  let userId = router.query.user || '';
+  if (Array.isArray(userId)) userId = userId[0];
+  userId = atob(userId);
+
+  let url = '';
+  if (userId) {
+    url = `/api/openstad/api/user/${userId}`;
+  }
+
+  let match = userId.match(/^(.+)-\*-(.+)$/);
+  if (match) {
+    url = `/api/openstad/api/user?&byIdpUser[identifier]=${match[2]}&byIdpUser[provider]=${match[1]}`
+  }
 
-  const userSwr = useSWR(userId ? url : null);
+  const userSwr = useSWR(url);
 
   async function updateUser(body: any) {
+
+    if (Array.isArray(body)) {
+      // update roles
+    } else {
```

Comment:
Moet hier nog iets gebeuren, of kunnen we de `if` omdraaien zodat de `else` vervalt?

Reply:
Goed punt, dat was een oudere gedachte die inmiddels achterhaald is